static/cockroach/v3-chinook-ndc-metadata/configuration.json

```diff
@@ -2803,19 +2803,19 @@
         "enum": ["hearts", "clubs", "diamonds", "spades"]
       },
       "char": "string",
-      "date": "string",
-      "float4": "number",
-      "float8": "number",
-      "int2": "integer",
-      "int4": "integer",
-      "int8": "integer",
-      "numeric": "number",
+      "date": "date",
+      "float4": "float32",
+      "float8": "float64",
+      "int2": "int16",
+      "int4": "int32",
+      "int8": "json",
```

Comment:
These are the changes to the configuration. Number and String are deprecated in ndc-spec. We will still accept these for backwards compatibility reasons, but we will output the new representations. @danieljharvey